Objects/AutoFillMultiRollerArena.ttslua

```diff
@@ -40,6 +40,24 @@ function onPlayerTurnEnd(player_color_end, player_color_next)
     _activeSystemGuid = false
 end
 
+function _applyPlanetDecals(system)
+    local decals = {}
+
+    for _, planet in ipairs(system.planets or {}) do
+        if planet.planetDecal then
+            table.insert(decals, {
+                name = planet.name .. ' decal',
+                url = planet.planetDecal,
+                position = { x = planet.position.x, y = 0.1, z = planet.position.z },
+                rotation = { x = 90, y = 180, z = 0 },
+                scale = { x = 2.3, y = 2.3, z = 1 }, -- TODO: rework Mirage image so this can be 1,1,1
```

Comment:
actually no, decal "scale" is not (this cost me a solid day the first time using decals).  decal "scale" is actually the world space size in which to fit the image, completely independent of the image itself.